### DISCUSSION TITLE
Can I run multiple scripts at same time

### DISCUSSION THREAD

**Discussion by Kiiperu** [Discord-OTP-Forcer/Discord-OTP-Forcer#140] [Community Support] [answered]
What if I run for the same account on multiple computers? Let's say I have 5 computers and I run the script on them at the same time. Will I get my IP blocked? Will the account be closed for further attempts? Will everything be ok? What happens?

**Comment by Inbydev** [comment, marked as answer]
Probably is gonna block your IP and they can delete or block your account for further attempts too.
